docs/File-Formats.md

```diff
@@ -1251,10 +1252,58 @@ The cells contain the p-value for the GSVA score: A real number, between 0.0 and
 
 <table>
 <thead><tr><th>geneset_id</th><th>TCGA-AO-A0J</th><th>TCGA-A2-A0Y</th><th>TCGA-A2-A0S</th></tr></thead>
-<tr><td>GO_POTASSIUM_ION_TRANSPOR</td><td>0.0811</td><td>0.0431</td><td>0.0087</td></tr>
+<tr><td>GO_POTASSIUM_ION_TRANSPORT</td><td>0.0811</td><td>0.0431</td><td>0.0087</td></tr>
 <tr><td>GO_GLUCURONATE_METABOLIC_PROCES</td><td>0.6621</td><td>0.0031</td><td>1.52e-9</td></tr>
 <tr><td>..</td><td></td><td></td><td></td></tr>
 </table>
 
 ## Study Tags file
 YAML or JSON file which contains extra information about the cancer study. No compulsory fields are required for this file (free-form). To enable this feature, you need to add a line in the cancer study meta file with `tags_file:` followed the YAML/JSON file name. The information on the YAML or JSON file will be displayed in a table when mousing over a tag logo in the studies on the query page.
+
+## Treatment Data
+Treatment response data relate to outcome variables of treatments (compounds or combinations thereof) on samples (e.g., cell lines). cBioPortal supports any number of response data types imported from separate data files. Information on treatments (`treatment_id`, `name`, `description` and `url`) are not part of the cBioPortal seed database, but are imported automatically from treatment response data files. For treatments already present in the database the `name`, `description` and `url` fields are overwitten by subsequent imports of treatment response data files. The `name`, `description` and `url` fields/columns are optional. The `treatment_id` and `name` fields must be unique within a data file. When the `name`, `description` and `url` fields are not defined in a data file, these fields will receive the value of the `id` field. When multiple treatment data files are loaded as part of study, the `treatment_id`, `name`, `description` and `url` columns must be identical is all data files.
+
+### Treatment response meta file
+The meta file will be similar to meta files of other genetic profiles, such as mRNA expression. For treatment response data `RESPONSE-VALUE` is used as `datatype`. Any number of treatment response parameters can be loaded by means of different data files. For treatment meta files values for `stable_id` are not pre-registered in the cBioPortal implementation, but can be freely chosen by the user. Requirement is that each treatment meta file is assigned an unique `stable_id`.
+
+Required fields: 
+```
+cancer_study_identifier: Same value as specified in study meta file
+genetic_alteration_type: TREATMENT_RESPONSE
+datatype: RESPONSE-VALUE
+stable_id: Any unique identifier within the study
```

Comment:
thinking about error scenarios...it would be nice if we say "any unique identifier using a combination of alphanumeric characters and _". This is the current format for other stable ids. I just don't know what could break if one would deviate from it now. So let's validate it as such.

Reply:
Docs were updates.

Reply:
please also add the validation to enforce it.